### DISCUSSION TITLE
Uninstall previous version?

### DISCUSSION THREAD

**Discussion by SahajGit** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#345] [Q&A] [answered]
I currently have AlDente v2.2 and want to install the latest v1.04. Do I need to uninstall the one I have before getting this one?
Also, if I need to uninstall, any setting I need to tweak other than setting max charge back to 100%?
Thanks for the help :)

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @SahajGit,

just set max charge back to 100% and wait a little bit. Afterward, you can uninstall/delete AlDente Classic 2.2 and start with the installation of AlDente 1.04. :)

**Reply by SahajGit** [reply]
@MatthiasKerbl Thanks a ton mate!
